src/Draft.js

```diff
@@ -53,6 +55,9 @@ const DraftPublic = {
   Modifier: DraftModifier,
   RichUtils: RichTextEditorUtil,
 
+  DefaultDraftBlock,
```

Comment:
I am also exposing now DefaultDraftInlineStyle for consistency and because we will need that once we make inline styles also configurable